### ISSUE TITLE
Pouvoir filtrer module signalétique par "Type de lames"

### ISSUE THREAD

**Issue by julienchaudet** [GeotrekCE/Geotrek-admin#3841]
Bonjour, je n'ai pas trouvé comment filtrer la signalétique par type de lame.
Merci d'avance

**Comment by AudreyRemy** [comment]
Bonjour, 
Je crois qu'on ne peut filtrer la signalétique que par type de signalétique et les lames par type de lames. 
Pour faire apparaitre la liste des lames il faut sélectionner la couche Lames
![Capture](https://github.com/GeotrekCE/Geotrek-admin/assets/78603108/091feb7e-d17a-413f-af13-e23c04c123e6)

**Comment by julienchaudet** [comment]
Oui, parfait, je n'avais pas pensé à sélectionner la couche "lames". Merci beaucoup
